PGM/src/main/java/tc/oc/pgm/polls/PollBlacklist.java

```diff
@@ -13,27 +17,30 @@
 import java.util.List;
 import java.util.Optional;
 
-public class PollableMaps {
+@Singleton
+public class PollBlacklist implements PluginFacet {
 
-    private List<PGMMap> maps;
+    private List<PGMMap> blacklistedMaps = new ArrayList<>();
```

Comment:
Just store the IDs to reduce the memory footprint.

Reply:
shouldn't make a difference unless the objects are being cloned